readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Take this example:

```js
const StyledComp = styled.div`
  ${props => props.complex ? complexMixin : 'color: blue;'}
`;
```
In case the result is static `color: blue;` its easy to analyze and covert to an ISTF property/value. But in case its a whatever `complexMixin` is and it could be generated in a way that is very hard to analyze or it could be even `complexMixin(props)` which in return does any complex calculations based on the moon position. 

This is insane.